source/custom/nixnet_converters.h

```diff
@@ -103,7 +104,7 @@ struct FrameHolder {
         throw std::invalid_argument("All FrameBufferRequest instances in repeated field should have same oneof set for the frame");
       }
 
-      size_t frame_size = ENET_FRAME_HEADER_LENGTH + grpc_frame.enet().frame_data().length();
+      size_t frame_size = ENET_FRAME_HEADER_LENGTH + ENET_FRAME_FCS_SIZE + grpc_frame.enet().frame_data().length();
```

Comment:
Can you call the helper to compute size instead? That way the logic will be single place.

```
get_frame_buffer_size(
  number_of_frames=1,
  max_payload_per_frame=grpc_frame.enet().frame_data().length(),
  protocol=Protocol::ENET`
);
```

Reply:
Done